dexs/sudo-fina/index.ts

```diff
@@ -0,0 +1,46 @@
+import fetchURL from "../../utils/fetchURL"
+import { Chain } from "@defillama/sdk/build/general";
+import { SimpleAdapter } from "../../adapters/types";
+import { CHAIN } from "../../helpers/chains";
+import { getUniqStartOfTodayTimestamp } from "../../helpers/getUniSubgraphVolume";
+
+type IUrl = {
+  [s: string]: string;
+}
+
+const url: IUrl = {
+  [CHAIN.SUI]: "https://e5370tyxcb.execute-api.ap-northeast-1.amazonaws.com/volume"
+}
+
+interface IVolume {
+  totalVolume: number,
+  dailyVolume: number,
+  weekVolume: number,
+  monthVolume: number,
+}
+
+const fetch = (chain: Chain) => {
+  return async (timestamp: number) => {
+    const dayTimestamp = getUniqStartOfTodayTimestamp(new Date(timestamp * 1000))
+    const volume: IVolume = (await fetchURL(url[chain]))?.data;
+    return {
+      totalVolume: `${volume?.totalVolume || undefined}`,
+      dailyVolume: `${volume?.dailyVolume || undefined}`,
+      weekVolume: `${volume?.weekVolume || undefined}`,
+      monthVolume: `${volume?.monthVolume || undefined}`,
+      timestamp: dayTimestamp,
+    };
+  };
+}
+
+const adapter: SimpleAdapter = {
+  adapter: {
+    [CHAIN.SUI]: {
+      fetch: fetch(CHAIN.SUI),
+      runAtCurrTime: true,
+      start: async () => 1682121600,
```

Comment:
did we make sure the start time is the first day we launched? 1/12 or 1/13 i think